components/ClipboardCopy.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+const ClipboardCopy = ({ copyText, children }) => {
```

Comment:
Do we need `copyText` and `children`? They look the same

Reply:
The `children` have the code blocks commas in the start and end. Should I use slice to remove the firsts 5 and the last three characters? Then We could probably use the `children` prop itself

Reply:
Oh good point 🤔 I think yes, because I am keen to try and use the same code, so there is not duplicate and not the risk of the code being different and causing bugs for people.

It would probably be safer to separate on the line breaks and remove the first line and last line - because it might not always be 5 characters at the beginning

Reply:
Is there a way to get the `children` as a string?

Reply:
Is there a way to convert the children object to string?

Reply:
> Oh good point 🤔 I think yes, because I am keen to try and use the same code, so there is not duplicate and not the risk of the code being different and causing bugs for people.
> 
> It would probably be safer to separate on the line breaks and remove the first line and last line - because it might not always be 5 characters at the beginning

Oh yeah, That is a way better option. Could you tell me how would I convert the `children` prop to string.  I can't seem to find the method to get the text from the object

Reply:
I haven't tried before, but after a Google I found this https://github.com/fernandopasik/react-children-utilities

Reply:
I'll give it a go, thanks!